helpers/FOLDER.py

```diff
@@ -103,6 +104,7 @@ def get_image_to_folder(basepath, filename, bbox, year, save_metadata=False, ove
     # we can mimick ESRI MapImageLayer's metadata, 
     # at least the section that we need
     image_metadata = {
+        "year": year,
```

Comment:
To prevent a bug due to 'year' value being filled with None in case no year is provided I propose to replace this line with:
`**({'year': year} if year else {}),`
then the key year will be added only if it exists. The then changes made l783 in `generate_tilesets.py` can be deleted.